### DISCUSSION TITLE
products expire date

### DISCUSSION THREAD

**Discussion by vicktor45** [hay-kot/homebox#479] [Ideas]
it is also possible to add an expiration date for the products perhaps with relative notification before they expire, it could be useful for managing the kitchen or pantry perhaps even adding more than one expiry date for each product.

example: I have 2 cans of tomatoes, one expires on 16/07 and I receive a notification on 10/07 while another one expires on 2/08 and on this other the notification will arrive a week before 

sorry for my english

**Comment by boubiro** [comment]
hello, I think this is a very good idea, it would allow us to manage food stock more efficiently and make meals based on foods whose dates are close.
